package.json

```diff
@@ -21,7 +21,7 @@
     "charm": "^1.0.2",
     "google-auth-library": "^0.10.0",
     "googleapis": "^16.0.0",
-    "lighthouse": "1.6.3",
+    "lighthouse": "paulirish/lighthouse#b7756033",
```

Comment:
temporary commit over on the paulirish fork to get compiled CLI files. 

i acknowledge the lack of a lighthouse release is pretty painful for pwmetrics at this point. :)